services.yaml

```diff
@@ -3184,12 +3174,21 @@ definitions:
       full_key:
         type: string
         example: S35817XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
-      role:
-        type: string
-        description: user's role in the environment
-        example: admin
-  TeamMembers:
-    description: List of environment key team members
+  TeamMembersBasic:
+    description: List of environment team members
+    type: array
```

Comment:
here as well, i would add some object composition

Reply:
In this case it's similar to viewfull and viewbasic - teambasic includes fields editable by admins, teamfull includes all generated and personal data that can't be edited by admin